src/views/tickets/rsvp.php

```diff
@@ -21,177 +21,177 @@
 $now = current_time( 'timestamp' );
 ?>
 
+<form
+	id="rsvp-now"
+	action=""
+	class="tribe-tickets-rsvp cart <?php echo esc_attr( $messages_class ); ?>"
+	method="post"
+	enctype='multipart/form-data'
+>
 	<h2 class="tribe-events-tickets-title tribe--rsvp">
 		<?php echo esc_html_x( 'RSVP', 'form heading', 'event-tickets' ) ?>
 	</h2>
 
-	<form
-		id="rsvp-now"
-		action=""
-		class="tribe-tickets-rsvp cart <?php echo esc_attr( $messages_class ); ?>"
-		method="post"
-		enctype='multipart/form-data'
-	>
-		<div class="tribe-rsvp-messages">
-			<?php
-			if ( $messages ) {
-				foreach ( $messages as $message ) {
-					?>
-					<div class="tribe-rsvp-message tribe-rsvp-message-<?php echo esc_attr( $message->type ); ?>">
-						<?php echo esc_html( $message->message ); ?>
-					</div>
-					<?php
-				}//end foreach
-			}//end if
-			?>
+	<div class="tribe-rsvp-messages">
+		<?php
+		if ( $messages ) {
```

Comment:
if this is a view; Can we make these tags more like the ones on the rest of our templates?
```php
<?php if ( condition ) : ?>
    <?php foreach ( $messages as $message ) : ?>
    <?php endforeach; ?>
<?php endif; ?>
```